lightning/src/ln/channel.rs

```diff
@@ -444,6 +444,9 @@ pub const COMMITMENT_TX_WEIGHT_PER_HTLC: u64 = 172;
 /// it's 2^24.
 pub const MAX_FUNDING_SATOSHIS: u64 = 1 << 24;
 
+/// Maximum counterparty `dust_limit_satoshis` allowed. 2 * standard dust threshold on p2wsh output
```

Comment:
Can you add some docs on how you derived the 330 part of this value? In Bitcoin Core I don't think there is a "standard p2wsh output dust threshold".